clients/src/main/resources/common/message/ShareFetchRequest.json

```diff
@@ -48,6 +47,8 @@
         "about": "The partitions to fetch.", "fields": [
         { "name": "PartitionIndex", "type": "int32", "versions": "0+",
           "about": "The partition index." },
+        { "name": "PartitionMaxBytes", "type": "int32", "versions": "0",
+          "about": "The maximum bytes to fetch from this partition. 0 when only acknowledgement with no fetching is required. See KIP-74 for cases where this limit may not be honored." },
```

Comment:
Query: Can't we remove it from Request itself?

Reply:
It was in v0 (AK 4.0).

Reply:
Yeah, so we won't remove it just deprecate 👍